### PR TITLE
Load all nodes before deploy to check integrety

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: conda_forge_tick/cli.xsh [hunk 1]
```diff
@@ -31,7 +31,10 @@ def deploy(args):
             print(e)
 
     try:
-        load_graph()
+        gx = load_graph()
+        # TODO: be more selective about which json to check
+        for node, attrs in gx.nodes.items():
+            attrs._load()
```
Comment (beckermr) [inline conda_forge_tick/cli.xsh:37]: How much time does this take on your machine? I want to make sure it is not an hour or something?

Comment (beckermr) [reply conda_forge_tick/cli.xsh:37]: Also this fails on my machine. No attr _load.

Comment (beckermr) [reply conda_forge_tick/cli.xsh:37]: ` attrs["payload"]._load()` works

it takes 5-10 seconds on my machine

Comment (CJ-Wright) [inline conda_forge_tick/cli.xsh:37]:
```suggestion
            attrs["payload"]._load()
```

**Comment by CJ-Wright** [comment]
Sorry for the bug, this should be fixed now @beckermr

**Pull request merged**